src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1852,6 +1852,69 @@ limitations under the License.
             <include cluster="Mode Select" client="false" server="true" clientLocked="false" serverLocked="true"></include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-room-airconditioner</name>
+        <domain>HA</domain>
+        <typeName>Matter Room Air Conditioner</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0072</deviceId>
+        <clusters lockOthers="true">
+            <include cluster="Identify" client="false" server="true" clientLocked="true" serverLocked="true">
+                <requireAttribute>IDENTIFY_TIME</requireAttribute>
+                <requireAttribute>IDENTIFY_TYPE</requireAttribute>
+                <requireCommand>Identify</requireCommand>
+                <requireCommand>IdentifyQuery</requireCommand>
+                <requireCommand>TriggerEffect</requireCommand>
```

Comment:
I don't see where this device type makes TriggerEffect mandatory.  This seems like a bad copy/paste from the lighting device types which _do_ make it mandatory.

Again, if other device types shouldn't make this mandatory but do, file issues.